prime-dotnet-webapi/Services/EmailDocumentsService.cs

```diff
@@ -71,6 +80,37 @@ public async Task SaveSiteRegistrationReview(int siteId, Pdf pdf)
             await _context.SaveChangesAsync();
         }
 
+        private async Task<Pdf> GenerateHealthAuthorityRegistrationReviewAttachmentAsync(int healthAuthoritySiteId)
+        {
+            var model = await _context.HealthAuthoritySites
+                .Where(has => has.Id == healthAuthoritySiteId)
+                .Select(has => new HealthAuthoritySiteSubmissionViewModel
+                {
+                    SiteName = has.SiteName,
+                    SiteAddress = has.PhysicalAddress,
+                    AuthorizedUserName = $"{has.AuthorizedUser.Party.FirstName} {has.AuthorizedUser.Party.LastName}",
+                    HealthAuthorityName = has.HealthAuthorityOrganization.Name,
+                    PEC = has.PEC,
+                    CareType = has.HealthAuthorityCareType.CareType,
+                    NewOrExisting = string.IsNullOrWhiteSpace(has.PEC) ? "New Site" : "Existing Site",
+                    Vendor = has.HealthAuthorityVendor.Vendor.Name,
+
+                    PharmaNetAdministrator = new HealthAuthoritySiteSubmissionViewModel.HealthAuthorityContactViewModel
+                    {
+                        JobTitle = has.HealthAuthorityPharmanetAdministrator.Contact.JobRoleTitle,
+                        FullName = $"{has.HealthAuthorityPharmanetAdministrator.Contact.FirstName} {has.HealthAuthorityPharmanetAdministrator.Contact.LastName}",
+                        Phone = has.HealthAuthorityPharmanetAdministrator.Contact.Phone,
+                        Fax = has.HealthAuthorityPharmanetAdministrator.Contact.Fax,
+                        SmsPhone = has.HealthAuthorityPharmanetAdministrator.Contact.SMSPhone,
+                        Email = has.HealthAuthorityPharmanetAdministrator.Contact.Email
+                    },
+                })
+                .SingleAsync();
```

Comment:
If you are feeling up to it, this is a mapping that would be better done using AutoMapper.

Reply:
Now using `AutoMapper`